src/Core/Models/Domain/PasswordGenerationOptions.cs

```diff
@@ -9,7 +9,7 @@ public PasswordGenerationOptions(bool defaultOptions)
             if (defaultOptions)
             {
                 Length = 14;
-                Ambiguous = false;
+                AvoidAmbiguous = false;
```

Comment:
I'm not a fan of this particular change since it's inconsistent with all the other vars in the model (i.e. bools are named so `true` means "use this", not "don't use this").  In addition by coupling the name with the expectation in the UI, it will forever require updating should we ever change the presentation (i.e. if we change the UI from "Avoid Ambiguous" to "Allow Ambiguous").  I think the original naming should remain.  Thoughts @fedemkr ?

Reply:
@mp-bw I usually follow that approach as well but in this particular case it is misleading because how we perceive the feature. In the UI the "true" is to avoid, and that was the cause of the bug in the first place.

Reply:
@andrebispo5 My thought process is that the data model should not be forced to bend to the UI paradigm, which could change at some point.  Changing the name in this way forces our data model to have a single inconsistently named var for the sake of the UI, forcing the dev to do mental gymnastics if/when using the model elsewhere, when the inversion should just take place in the ViewModel, since the UI is the only place where inversion is required.

Reply:
I hadn't thought about it but I agree with @mp-bw we should be consistent model wise. Maybe we could take the UX name and name it as `AllowAmbiguous` in the model so to avoid confusion, because just the word `Ambiguous` could be misleading.
And even though it could be a fraction of a thinking, having not operator with a negative variable is less readable than not operator with positive name: `!AvoidAmbiguous` vs `!AllowAmbiguous`; furthermore the worst would be `!NotAllowAmbiguous`

Reply:
@fedemkr ye maybe changing the variable name to AllowAmbiguous might help with readability, what do you think @mp-bw? i think is the best trade-off overall.

Reply:
I'm good with `AllowAmbiguous`